MetaCocktailsSwiftData/Model/Cocktails/MiscModernCocktails/WhiteLinen.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  WhiteLinen.swift
+//  MetaCocktailsSwiftData
+//
+//  Created by James Menkal on 8/17/24.
+//
+
+import Foundation
+
+var whiteLinen = Cocktail(cocktailName: "White Linen",
+                          imageAsset: nil,
+                          glasswareType: .collins,
+                          garnish: [.cucumberSlices],
+                          ice: .koldDraft,
+                          author:Author(person: AuthorNames.reneDominguez.rawValue,
+                                        place: AuthorPlaces.ellaDiningRoom.rawValue,
+                                        year: "2008"),
+                          spec: whiteLinenSpec,
+                          buildOrder: whiteLinenBuild,
+                          tags: whiteLinenTags)
+
+let whiteLinenSpec: [OldCocktailIngredient] = [OldCocktailIngredient(.soda(.sodaWater), value: 2),
+                                               OldCocktailIngredient(.fruit(.cucumberSlices), value: 6, unit: .gentlyMuddled),
+                                               OldCocktailIngredient(.juices(.lime), value: 0.75),
+                                               OldCocktailIngredient(.syrups(.richSimple), value: 0.34, prep: PrepBible.richSimple),
+                                               OldCocktailIngredient(.liqueurs(.giffardElderflour), value: 0.5),
```

Comment:
If we're staying true to the original recipe, we should have St Germain instead of Giffard here right?